.circleci/config.yml

```diff
@@ -65,8 +65,11 @@ jobs:
   test-browser:
     <<: *test-defaults
     environment:
-      - TEST_TARGET=browser
-      - EMSCRIPTEN_BROWSER=0
+      # browser.test_sdl_ttf_render_text_solid depends on Arial from Microsoft
+      - TEST_TARGET=browser skip:browser.test_sdl_ttf_render_text_solid
+      - TEST_DEPENDENCY=firefox
+      - EMSCRIPTEN_BROWSER=firefox -headless
+      - EMSCRIPTEN_LACKS_HARDWARE_ACCESS=1
```

Comment:
looking at our other test suite env vars, I think the prefix `EM_` is more consistent with them.